adobe-esign-connector/src/com/axonivy/connector/adobe/esign/connector/service/AdobeSignService.java

```diff
@@ -289,6 +363,18 @@ public String uploadDocument(UploadWrapper upload) throws BpmError {
 
 		return documentId;
 	}
+	
+	/**
+	 * Wrapper method to call sub process to upload new transient documents required to create a new agreement
+	 *
+	 * @param upload
+	 * @return
+	 * @throws BpmError
+	 */
+	public List<String> uploadDocuments(List<UploadWrapper> upload) throws BpmError {
+		List<String> result = upload.stream().map(u -> uploadDocument(u)).collect(Collectors.toList());
+		return result;
```

Comment:
can we directly return the `upload.stream().map(u -> uploadDocument(u)).collect(Collectors.toList())` ? instead of create a local var then return it?